cypress/e2e/course_management/showCoursesInTable.feature

```diff
@@ -0,0 +1,98 @@
+@stable_test
+@regression_test
+@school_api_test
+Feature: Course - To show courses in a table with respective functionality
+
+    As an administrator I want to see all courses belonging to my school.
+
+    Scenario Outline: As an admin i can add a course to school
+        Given I am logged in as a '<user>' at '<state>'
+        When I click on administration in menu
+        When I go to new course administration page
```

Comment:
the wording is not optmial, i give you that ^^ But the step is not identical... the navigational steps designed by qa also have a url validation and he url is definetly different. I could remove the new for the new page and add a "legacy" to the old one. this way a refactoring is not needed and the renaming is not too big, because the legacy page was  not used in any test before this one 😅